app/presenters/summary/assessment_question_presenter.rb

```diff
@@ -0,0 +1,83 @@
+module Summary
+  class AssessmentQuestionPresenter < SummaryPresenter
+    include PresenterHelpers
+
+    def label
+      t("summary.section.questions.#{section_name}.#{name}")
+    end
+
+    def answer
+      return default_answer unless belongs_to_group?
+      case parent_group_answer
+      when 'unknown', nil
+        error_content('Missing')
+      when 'no', false
+        'No'
+      else
+        answer = answer_value(value)
+        relevant_answer? ? highlight(answer) : answer
+      end
+    end
+
+    def details
+      return unless has_dependencies?
+      dependency_questions.each_with_object([]) do |dependency, output|
+        if dependency.complex?
+          output << complex_detail_context(dependency)
+        elsif public_send(dependency.name).present?
+          output << detail_content(dependency.name)
+        end
+      end.join('. ')
+    end
+
+    private
+
+    def default_answer
+      case value
+      when 'unknown', nil
+        error_content('Missing')
+      when 'no', false
+        'No'
+      when 'yes', true
+        highlight('Yes')
+      else
+        answer = answer_value(value)
+        relevant_answer? ? highlight(answer) : answer
+      end
+    end
+
+    def detail_content(attribute)
+      [detail_label(attribute), answer_value(public_send(attribute))].join('')
+    end
+
+    def complex_detail_context(question)
+      public_send(question.name).each_with_object([]) do |item, output|
+        output << question.subquestions.map do |subquestion|
+          [
+            detail_label("#{question.name}_collection.#{subquestion.name}"),
+            answer_value(item.send(subquestion.name))
+          ].join(' ')
+        end.join(' | ')
+      end.join('</br>')
+    end
+
+    def detail_label(attribute)
+      I18n.t!(attribute, scope: [:summary, :section, :questions, section_name])
```

Comment:
I think we can get rid of I18n as we are including PresentersHelpers that include ActionView::Helpers::TranslationHelper